lib/events/auditlog.go

```diff
@@ -249,22 +278,42 @@ func (l *AuditLog) migrateSessions() error {
 	return nil
 }
 
-// PostSessionChunk writes a new chunk of session stream into the audit log
-func (l *AuditLog) PostSessionChunk(namespace string, sid session.ID, reader io.Reader) error {
-	if namespace == "" {
+// PostSessionChunks writes a new chunk of session stream into the audit log
+func (l *AuditLog) PostSessionSlice(slice SessionSlice) error {
+	if slice.Namespace == "" {
 		return trace.BadParameter("missing parameter Namespace")
 	}
-	sl, err := l.LoggerFor(namespace, sid)
+	if len(slice.Chunks) == 0 {
```

Comment:
LoggerFor opens file, so did not want to open a file on server if we know it won't work